lib/matplotlib/text.py

```diff
@@ -1380,6 +1386,30 @@ def set_fontname(self, fontname):
         """
         self.set_fontfamily(fontname)
 
+    def ha_for_angle(self, angle):
```

Comment:
I think these two new functions are best privatized (prefix the function name with `_`).